turtle.js

```diff
@@ -0,0 +1,138 @@
+// worked on it on Monday Nov 19, 2018 at 930pm
+class Turtle {
+
+    constructor (x, y){
+        this.x = x;
+        this.y = y;
+        this.face = 'east';
+        this.coordinates = [];
+        this.coordinates.push([this.x, this.y])
+        
+   }
+
+   allPoints(){
+       
+         return this.coordinates
+        
+   }
+   
+   forward(steps) {
+
+     if (this.face === 'east'){
+          for (let i = 0; i < steps; i++) {
+          this.x += 1;
+          this.coordinates.push([this.x, this.y])
+          }
+     } else if (this.face === 'south') {
+          for (let i = 0; i < steps; i++) {
+          this.y += 1;   
+          this.coordinates.push([this.x, this.y])
+          }
+     } else if (this.face === 'west') {
+          for (let i = 0; i < steps; i++) {
+          this.x += -1;
+          this.coordinates.push([this.x, this.y])
+          }
+     } else if (this.face === 'north') {
+          for (let i = 0; i < steps; i++) {
+          this.y += -1
+          this.coordinates.push([this.x, this.y])
+          }
+     }
+   }
+   right(){
+        if (this.face === 'east') {
+             this.face = 'south'
+        } else if (this.face === 'south') {
+             this.face = 'west'
+        } else if (this.face === 'west') {
+             this.face = 'north'
+        } else if (this.face === 'north') {
+             this.face = 'east'
+        }
+     }
+
+   left(){
+     if (this.face === 'east') {
+          this.face = 'north'
+     } else if (this.face === 'north') {
+          this.face = 'west'
+     } else if (this.face === 'west') {
+          this.face = 'south'
+     } else if (this.face === 'south') {
+          this.face = 'east'
+     }
+   }  
+
+   print(){
+     let allCoordinates = this.coordinates;
+     let maxCoordinates = [];
+     let board = [];
+     let boardString = '';
+ 
+     function findMax (x){
```

Comment:
It is helpful to give your variables names that are descriptive.
Here `x` is confusing. With a name like `x`, given the context of the problem, it is easy for one to assume it has something to do with an x-coordinate, especially since we are using `findMax` to the find the maximum x- and y-coordinates within the array `allCoordinates`.
A better, more descriptive name for the parameter would have been `coordinateArray` or something like that

Reply:
Similarly, `findMax` could have been named a little better to something like `findMaxXY` or `findMaxXYCoordinates` which one would assume would return a value like 
```js
return [biggestX, biggestY];
```

And then you would use it as follows on line 90:
```js
const maxCoords = findMaxXYCoordinates(allPoints);
```
and not you could use `maxCoords` within your print and get rid of your `maxCoordinates` array of length 1.